client/client.js

```diff
@@ -1,162 +1,153 @@
 // some layouts
 function setup_tabs() {
-    $("#codetabs").tabs({
+    $('#codetabs').tabs({
         activate: function (event, ui) {
         }
     });
-    
-    $("#outputtabs").tabs({
+
+    $('#outputtabs').tabs({
         activate: function (event, ui) {
-            var active = $("#outputtabs").tabs('option', 'active')
+            const active = $('#outputtabs').tabs('option', 'active');
             if (active == 0)  {
-                document.querySelectorAll("a[href='#console']")[0].style.color = "black";
+                document.querySelectorAll('a[href="#console"]')[0].style.color = 'black';
             } else if (active == 1) {
-                document.querySelectorAll("a[href='#output']")[0].style.color = "black";
+                document.querySelectorAll('a[href="#output"]')[0].style.color = 'black';
             } else if (active == 2) {
-                document.querySelectorAll("a[href='#tcpdump']")[0].style.color = "black";
+                document.querySelectorAll('a[href="#tcpdump"]')[0].style.color = 'black';
             }
         }
-    });    
+    });
 }
 
 function get_console_size() {
-    console_window = document.getElementById("console");
-    cols = Math.floor(console_window.offsetWidth / 10);
-    rows = Math.floor(console_window.offsetHeight / 19);
+    console_window = document.getElementById('console');
+    const cols = Math.floor(console_window.offsetWidth / 10);
+    const rows = Math.floor(console_window.offsetHeight / 19);
 
-    return {cols: cols, rows: rows};
+    return { cols, rows };
 }
 
 function on_window_resize() {
-    cols, rows = get_console_size();
+    const { cols, rows } = get_console_size();

```

Comment:
Seems curly braces for destructuring are missing here. Also `const` is needed to prevent these from being assigned globally.